controllers/movieController.js

```diff
@@ -0,0 +1,96 @@
+const Movie = require("../models/Movies");
+
+// List all movies from MongoDB database
+exports.movies_list = function(req, res, next) {
```

Comment:
JavaScript naming convention uses camelCase to name functions and variables, so I'd put `moviesList` and stick with that. Only use the lower score for constants, like:
```
const NUMBER_OF_MOVIES = 3;
```
PS. Keep in mind that not all `const` are CONSTANTS.

Reply:
It's done.  TY